File: cardbattlemodule/src/main/java/com/syahputrareno975/cardbattlemodule/model/playerBattleResult/PlayerBattleResultModel.java
package com.syahputrareno975.cardbattlemodule.model.playerBattleResult;

import com.syahputrareno975.cardbattlemodule.model.player.PlayerModel;
import java.io.Serializable;

public class PlayerBattleResultModel implements Serializable {

    public PlayerModel Owner = new PlayerModel();
    public Long DamageReceive  = 0L;
    public Long EnemyAtk = 0L;
    public Long OwnerDef = 0L;

    public PlayerBattleResultModel() {
    }

    public PlayerBattleResultModel(PlayerModel owner, Long damageReceive, Long enemyAtk, Long ownerDef) {
        Owner = owner;
        DamageReceive = damageReceive;
        EnemyAtk = enemyAtk;
        OwnerDef = ownerDef;
    }
}

File: app/src/main/java/com/syahputrareno975/simplecardbattle/model/playerBattleResult/PlayerBattleResultModel.kt
package com.syahputrareno975.simplecardbattle.model.playerBattleResult

import java.io.Serializable
import cardBattle.CardBattle.player
import com.syahputrareno975.simplecardbattle.model.player.PlayerModel


class PlayerBattleResultModel : Serializable {
    var Owner = PlayerModel()
    var DamageReceive: Long = 0
    var EnemyAtk: Long = 0
    var OwnerDef: Long = 0

    constructor()
    constructor(Owner: PlayerModel, DamageReceive: Long, enemyAtk: Long, ownerDef: Long) {
        this.Owner = Owner
        this.DamageReceive = DamageReceive
        this.EnemyAtk = enemyAtk
        this.OwnerDef = ownerDef
    }


}